source/scripting_v3/GTA/Game.cs

```diff
@@ -551,9 +551,10 @@ public static int GetProfileSetting(int index)
 		/// Searches the address space of the current process for a memory pattern.
 		/// </summary>
 		/// <param name="pattern">The pattern.</param>
+		/// <param name="startAddress">The address to start searching at. If <see cref="IntPtr.Zero" /> (<see langword="default" />), search is started at the base address.</param>
 		/// <returns>The address of a region matching the pattern, or <see cref="IntPtr.Zero" /> if none was found.</returns>
 		/// <remarks>This function takes the Cheat Engine/IDA format ("48 8B 0D ?? ?? ? ? 44 8B C6 8B D5 8B D8" for example, where ?? and ? are wildcards).</remarks>
-		public static IntPtr FindPattern(string pattern)
+		public static IntPtr FindPattern(string pattern, IntPtr startAddress = default)
```

Comment:
Won't this break binary compatibility with existing scripts using this method (since it's public)?

Reply:
The method is not exposed in any released versions (see [Game.cs for v3 API as of v3.3.2](https://github.com/crosire/scripthookvdotnet/blob/v3.3.2/source/scripting_v3/GTA/Game.cs)), so this change won’t break the compatibility.